tools/coverage/main.go

```diff
@@ -48,16 +49,17 @@ func main() {
 	artifactsDir := flag.String("artifacts", defaultArtifactsDir, "directory for artifacts")
 	coverageTargetDir := flag.String("cov-target", defaultCoverageTargetDir, "target directory for test coverage")
 	coverageProfileName := flag.String("profile-name", defaultCoverageProfileName, "file name for coverage profile")
+	noRepoConn := flag.Bool("no-repo-connection", true, "enable it when connection to GibHub is not required")
```

Comment:
why is this true by default? I think this should default to false and we should assume github by default. Also, rename this flag to something like `isGithub`

Reply:
removed in https://github.com/knative/test-infra/pull/1425